cmd/protoc-gen-go-grpc/README.md

```diff
@@ -14,7 +14,8 @@ To restore this behavior, set the option `require_unimplemented_servers=false`.
 E.g.:
 
 ```sh
-  protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false[,other options...] \
+  protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false \
```

Comment:
@hanut19 did you try this? Its giving error. All grpc options actually should be provided in same flag. It has to be something like protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false[,opt2=value2,opt3=value3,etc...]